CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Change Log
 
 ## [Unreleased]
+### Added
+- Security - Protect the liana by adding an IP Whitelist.
```

Comment:
- Premium Security - Users that won't use an authorized IP won't have access to the data.